logxhost/log.go

```diff
@@ -0,0 +1,47 @@
+package logxhost
+
+import (
+	"time"
+
+	"github.com/Masterminds/squirrel"
+	"github.com/jmoiron/sqlx"
+	dbutil "github.com/monstercat/golib/db"
+)
+
+type Log struct {
+	Id      string `setmap:"ignore"`
+	Machine string
+	Service string
+	Context string
+	Message string
+	LogType string    `db:"log_type"`
+	LogTime time.Time `db:"log_time"`
+	Created time.Time
+}
+
+var (
+	ColsLogs = dbutil.GetColumnsList(&Log{}, "")
+)
+
+func BuildSelectLogs() squirrel.SelectBuilder {
+	return psql.Select(ColsLogs...).From(TableLogs)
```

Comment:
You can do this as a var so it doesn't need to be recreated on package level
`var SelectLogQry = psql....`